osu.Game/Rulesets/Objects/Drawables/DrawableHitObject.cs

```diff
@@ -630,7 +598,7 @@ protected override void UpdateAfterChildren()
 
         /// <summary>
         /// The time at which state transforms should be applied that line up to <see cref="HitObject"/>'s StartTime.
-        /// This is used to offset calls to <see cref="UpdateStateTransforms"/>.
+        /// This is used to offset calls to <see cref="UpdateHitStateTransforms"/>.
```

Comment:
```suggestion
        /// This is used to offset calls to <see cref="UpdateStartTimeStateTransforms"/>.
```

Reply:
Should we update the one above as well? Not even sure which that should mention (both?).